docs/changehistory/NextVersion.md

```diff
@@ -35,10 +36,24 @@ Table of contents:
 - `PropertyFilterBuilderToolbar` component to display the action buttons in the filter builder. #686
 - `isGroupOperatorDisabled` property of `PropertyFilterBuilderRendererProps` to control whether the group operator is toggle-able. #686
 
+### FilterBuilder changes
```

Comment:
Make sure the PR number of your change is referenced.
Follow the `NextVersion.template.md`: instead of a new section, just indent your list if needed (I'd rather not mix 686 in there):
```
- ... #686
- Added support ... #702
  - Changed ...
  - Updated ...
```